### DISCUSSION TITLE
touched the volume control on my monitor, now I can't control the volume with monitor control

### DISCUSSION THREAD

**Discussion by totalktoapropress1** [MonitorControl/MonitorControl#1429] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

I used a new usb-c to hdmi cable because the old cable sometimes caused the screen to black out. I was able to control the volume via monitorcontrol until I touched the volume control on my monitor. Now a change on the volume in monitorcontrol doesn't change the volume of the monitor. The brightness control still works. I switched back to my old cable and it worked just fine.

### Expected behavior

A change in the volume on monitor control directs a change in the volume on the monitor

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.5
- Mac model: 13' M2 MacBook Air
- MonitorControl version: 4.1.0
- Monitor(s): LG 27UL500-W
- Monitor Cable(s)/Connection(s): usb-c - hdmi
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
If it works with an other cable then this seems to be a hardware related issue.
